src/sidebar/services/annotations.js

```diff
@@ -9,14 +8,26 @@ import { generateHexString } from '../util/random';
 import uiConstants from '../ui-constants';
 
 // @ngInject
-export default function annotationsService(
-  annotationMapper,
-  api,
-  store,
-  streamer,
-  streamFilter
-) {
-  let searchClient = null;
+export default function annotationsService(api, store) {
```

Comment:
Can you add a comment above this to clarify the responsibilities of this service vs. `loadAnnotationsService`, and the same in that service?